lint/lint.py

```diff
@@ -226,6 +226,23 @@ def check_parsed(repo_root, path, f, css_mode):
     if source_file.type == "visual" and not source_file.name_is_visual:
         return [("CONTENT-VISUAL", "Visual test whose filename doesn't end in '-visual'", path, None)]
 
+    for reftest_node in source_file.reftest_nodes:
+        href = reftest_node.attrib.get("href", "")
+        path_parts = [repo_root]
+        if href[0] != "/":
+            path_parts.extend([source_file.dir_path, href])
+        else:
+            # Remove the leading "forward-slash" character in root-relative
+            # URLs so that the `os.path.join` does not interpret the value as
+            # an absolute path.
+            path_parts.extend([href[1:]])
+        reference_file = os.path.join(*path_parts)
+        reference_rel = reftest_node.attrib.get("rel", "")
+
+        if not os.path.isfile(reference_file):
+            return [("NON-EXISTENT-REF",
```

Comment:
Works for me. See commit https://github.com/w3c/wpt-tools/pull/159/commits/035126f9b4c1cffeb2882f6d051c2ffb3e45f5e2